core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -205,6 +205,33 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
 
   <script src="{{TEMPLATE_DIR_PREFIX}}/expressions/expressionInterpolationService.js"></script>
 
+  <!-- these are the SchemaBased files that were moved from base index.html-->
+  <script src="{{TEMPLATE_DIR_PREFIX}}/components/forms/schema_editors/SchemaBasedBoolEditorDirective.js"></script>
```

Comment:
I thought that too but when you remove them looks like some of the question would not be answered. I tried on welcome exploration and you actually can not see the  input field to type the answer if you remove them. Unless I am missing something.